content/en/docs/examples.md

```diff
@@ -59,7 +59,7 @@ For a more comprehensive set of examples, see the full rules file at `falco_rule
 
 - rule: write_binary_dir
   desc: an attempt to write to any file below a set of binary directories
-  condition: evt.dir = < and open_write and not package_mgmt_binaries and bin_dir
+  condition: evt.dir = < and open_write and not evt.dir in (package_mgmt_binaries) and bin_dir
```

Comment:
I think we want to have `proc.name` here instead of `evt.dir`. `evt.dir` refers to the direction of the event, and can only be one of `>` or `<`.

Reply:
Thank you! Agreed, made the changes